config/initializers/new_framework_defaults_7_0.rb

```diff
@@ -11,10 +11,10 @@
 
 # `button_to` view helper will render `<button>` element, regardless of whether
 # or not the content is passed as the first argument or as a block.
-# Rails.application.config.action_view.button_to_generates_button_tag = true
+Rails.application.config.action_view.button_to_generates_button_tag = true
```

Comment:
This may not be safe _but_ all of our `button_to` is in the admin panel. So if anything looks wrong there, I'll refactor some codes

Reply:
@jaw6 I know we had a to `button_to` or not to `button_to` conversation; do you have thoughts on this?

Reply:
```
$ rg "<%=? button_to" --files-with-matches
app/views/moderations/mod.html.erb
app/views/admin/profile_fields/_ungrouped_profile_fields.html.erb
app/views/admin/profile_fields/_grouped_profile_fields.html.erb
app/views/admin/users/index/_invitation_actions_dropdown.html.erb
app/views/admin/users/show/overview/_roles.html.erb
app/views/admin/users/show/profile/actions/_export.html.erb
```

Reply:
I briefly navigated around admin and it looks like nothing changed.

Reply:
In general, I'm maybe just less fond of `button_to` than I used to be. I don't think it's _bad_, it's more that I often find it obscures and complicates more than it helps, compared to "just write HTML" alternatives. (It isn't even consistently less verbose.)

It looks like this was commented-out prior and now we're explicitly setting it? Unless there's another config somewhere else that I'm missing, this should be a non-event. 

Even if it is a change, I think changing to the new Rails default is the way to go. We're using it in six places. It looks to me that five of six use the string form -- the form that changed, subtly, in Rails 7. Only one (`app/views/admin/users/show/overview/_roles.html.erb`) is using the block form.

So, prior to Rails7, the five `button_to 'String Here'` were emitting something like:

```html
<form>
  <input value="String Here" type="submit" />
</form>
```

And are now emitting:

```html
<form>
  <button type="submit">String Here</button>
</form>
```

(Again, this is a clear improvement, in my view -- `button_to` really _ought to_ make a **button**.) 

The potential impact here would be if we have front-end code that attaches specifically to these being `input`.

I'll poke around, but I expect this is fine.

Reply:
Thanks for the input @jaw6 !